var/spack/repos/builtin/packages/libxkbcommon/package.py

```diff
@@ -15,17 +17,59 @@ class Libxkbcommon(AutotoolsPackage):
     homepage = "https://xkbcommon.org/"
     url      = "https://xkbcommon.org/download/libxkbcommon-0.8.2.tar.xz"
 
-    version('0.8.2', sha256='7ab8c4b3403d89d01898066b72cb6069bddeb5af94905a65368f671a026ed58c')
-    version('0.8.0', sha256='e829265db04e0aebfb0591b6dc3377b64599558167846c3f5ee5c5e53641fe6d')
-    version('0.7.1', sha256='ba59305d2e19e47c27ea065c2e0df96ebac6a3c6e97e28ae5620073b6084e68b')
+    version('1.4.0', sha256='106cec5263f9100a7e79b5f7220f889bc78e7d7ffc55d2b6fdb1efefb8024031')
+    version('0.8.2', sha256='7ab8c4b3403d89d01898066b72cb6069bddeb5af94905a65368f671a026ed58c', deprecated=True)
+    version('0.8.0', sha256='e829265db04e0aebfb0591b6dc3377b64599558167846c3f5ee5c5e53641fe6d', deprecated=True)
+    version('0.7.1', sha256='ba59305d2e19e47c27ea065c2e0df96ebac6a3c6e97e28ae5620073b6084e68b', deprecated=True)
+
+    variant('wayland', default=False, description='Enable Wayland support')
 
+    depends_on('meson', type='build', when='@0.9:')
```

Comment:
This is already added by the base class automatically for all versions, you can remove this line